data/board.ts

```diff
@@ -0,0 +1,83 @@
+export type SocialMap = {
+  mastodon?: string
+}
+
+export type BoardMember = {
+  name: string
+  position: string
+  title?: string
+  otherTitle?: string
+  socials?: SocialMap
+  slug?: string
+  avatar?: string
+}
+
+export const BOARD_POSITION_DIRECTOR = "Board Director";
+export const BOARD_POSITION_OBSERVER = "Board Observer";
+
+export const board: BoardMember[] = [
```

Comment:
I kept board member data separate from the team member data (`data/team.ts`). Meant a bit of copying. I can merge the team and board member data if there is a strong desire to do so.